extensions/amp-mathml/validator-amp-mathml.protoascii

```diff
@@ -0,0 +1,69 @@
+#
```

Comment:
Please take a look at [Contributing Validator Rules](https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md). The format for allowing the script tag has been simplified greatly.

Reply:
Ok, will do - thanks for the feedback.

Reply:
I fixed the extensions/amp-mathml/validator-amp-mathml.protoascii file in https://github.com/ampproject/amphtml/pull/12836/commits/f6d93f3a5b14cdc926274eeafdc36405f11d4c6d - I still need to add some valid and invalid test cases